core/lib/env_config/src/proof_data_handler.rs

```diff
@@ -4,12 +4,22 @@ use crate::{envy_load, FromEnv};
 
 impl FromEnv for ProofDataHandlerConfig {
     fn from_env() -> anyhow::Result<Self> {
-        envy_load("proof_data_handler", "PROOF_DATA_HANDLER_")
+        Ok(Self {
+            http_port: std::env::var("PROOF_DATA_HANDLER_HTTP_PORT")?.parse()?,
+            proof_generation_timeout_in_secs: std::env::var(
+                "PROOF_DATA_HANDLER_PROOF_GENERATION_TIMEOUT_IN_SECS",
+            )?
+            .parse()?,
```

Comment:
AFAICT, this works:

```rust
Ok(Self {
    tee_config: envy_load("proof_data_handler.tee", "PROOF_DATA_HANDLER_")?,
    ..envy_load("proof_data_handler", "PROOF_DATA_HANDLER_")?
})
```

Reply:
[Fixed](https://github.com/matter-labs/zksync-era/pull/3112/commits/a75a0a2c7b8b86fd972e5364619c61f4e482fe1b). ✅